Formula/hound.rb

```diff
@@ -0,0 +1,17 @@
+require "formula"
+
+class Hound < Formula
+  homepage "https://github.com/thoughtbot/hound-cli"
+  url "https://github.com/thoughtbot/hound-cli/archive/v0.1.0.tar.gz"
+  version "0.1.0"
+  sha256 "88486a5151be3c0be0515111781b7be5a2f4621e38b1d211d0c3671ad78f7e34"
+
+  def install
+    prefix.install "lib" => "lib"
```

Comment:
you might want to double check how this copies data into `/usr/local/lib`. I'm assuming it doesn't overwrite that directory, but I think this will either create (maybe overwrite?) `/usr/local/lib/lib` _or_ it will just dump the contents directly into `/usr/local/lib`. Neither of these things seem like what you really intend.